programs/pkey/rsa_verify_pss.c

```diff
@@ -61,11 +61,22 @@ int main( int argc, char *argv[] )
     FILE *f;
     int ret = 1;
     size_t i;
+#if defined(MBEDTLS_PLATFORM_C)
+    mbedtls_platform_context platform_ctx;
+#endif
     mbedtls_pk_context pk;
     unsigned char hash[32];
     unsigned char buf[MBEDTLS_MPI_MAX_SIZE];
     char filename[512];
 
+#if defined(MBEDTLS_PLATFORM_C)
+    if( ( ret = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  !  mbedtls_platform_setup returned %d\n\n", -ret );
```

Comment:
The return value is displayed as a negative integer here, but as positive at line 136. We should be self-consistent within the same program.